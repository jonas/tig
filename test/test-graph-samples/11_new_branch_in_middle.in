commit A B C E
    Commit A - merge B, C, and E
commit B F
    Commit B - after F
commit C F
    Commit C - after F
commit E Z
    Commit E - after Z
commit F G I
    Commit F - merge G and I
commit G H I
    Commit F - merge H and I
commit H I
    Commit H - after I
commit I Z
    Commit I - after Z
commit Z
    Commit Z
