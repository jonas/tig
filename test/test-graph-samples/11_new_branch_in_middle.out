●─┬─╮ Commit A - merge B, C, and E
● │ │ Commit B - after F
│ ● │ Commit C - after F
│ │ ● Commit E - after Z
●─╯ │ Commit F - merge G and I
●─│─│─╮ Commit F - merge H and I
● │ │ │ Commit H - after I
●─┴─│─╯ Commit I - after Z
◎───╯ Commit Z
